### ISSUE TITLE
Possible to keep a running record of job postings to see additional dates? Or expire postings if not updated?

### ISSUE THREAD

**Issue by slushieken** [scotthasbrouck/HackerSurf#15]
A job posting will often have several dates tied to it.
1. Creation date: the date the posting first appeared.
2. Modified date - last time the posting was updated
3. Repost date - the last time posting was reposted

This helps to sort out reliable available jobs versus those have become stale as the posting has not been taken down yet.

Some sites like Indeed repost an existing job over and over. I am sure this is because the original poster did not make it around to deleting it when it is not needed any longer. The problem is that you cannot tell on Indeed if a searched posting is an actually 'New' posting made several hours ago by a human, or if it was just auto-posted by the engine. I know they probably do this to boost the numbers of postings, but it is a major pain to sort those out.

**Comment by scotthasbrouck** [comment]
Great point.
Yes, that data is generally available on almost every job posting site I've scraped. It just gets messy because usually that information is in the actual job page itself, not the listing page. So rather than scraping on the order of xx pages, you have to scrape xxxx pages.

Perhaps a better way: rather than scraping the dates off the pages them selves, is to have the app keep track of when the post first showed up, when it was modified, and when it was reposted. Then it would be straightforward to show a "freshness" gauge, like a cell phone signal bar meter, next to every post. That would be totally unique because it would be normalized across all job post websites.

Thoughts on that method?

**Comment by slushieken** [comment]
Actually, what you said Scott was exactly what I meant. I like it. Can it be done?

**Comment by scotthasbrouck** [comment]
Yes, I would be easier to do than scraping everything IMHO. I really like this feature, as it gives way more insight than just a post date.
I think I'll work on this next...
